src/lib/is_array.js

```diff
@@ -38,8 +38,15 @@ function isArray1D(a) {
     return !isArrayOrTypedArray(a[0]);
 }
 
+function isPrimitiveTypedArrayRepr(a) {
+    return (a !== undefined && a !== null &&
```

Comment:
`Lib.isPlainObject` should get rid of the first three checks.

Reply:
Done in 5030d3a